app/identifiers/register/establishers/company/IsCompanyDormantId.scala

```diff
@@ -16,15 +16,48 @@
 
 package identifiers.register.establishers.company
 
-import identifiers.TypedIdentifier
-import identifiers.register.establishers.EstablishersId
+import identifiers._
+import identifiers.register.establishers.{EstablishersId, IsEstablisherNewId}
+import models.Link
 import models.register.DeclarationDormant
+import models.register.DeclarationDormant._
+import play.api.i18n.Messages
 import play.api.libs.json.JsPath
+import utils.{Enumerable, UserAnswers}
+import utils.checkyouranswers.{CheckYourAnswers, IsDormantCYA}
+import viewmodels.AnswerRow
 
 case class IsCompanyDormantId(index: Int) extends TypedIdentifier[DeclarationDormant] {
   override def path: JsPath = EstablishersId(index).path \ IsCompanyDormantId.toString
 }
 
-object IsCompanyDormantId {
+object IsCompanyDormantId extends Enumerable.Implicits {
+
   override def toString: String = "isCompanyDormant"
```

Comment:
Dormant pages are bypassed in Variations journey, hence this code is not required

Reply:
reverted